.clang-format

```diff
@@ -28,7 +27,7 @@ BreakBeforeBraces: Custom
 BreakInheritanceList: BeforeComma
 BreakBeforeTernaryOperators: false
 BreakStringLiterals: false
-ColumnLimit:     109
+ColumnLimit: 0
```

Comment:
Hmm, are those changes to `.clang-format` by mistake?

Reply:
My bad, good catch!